### DISCUSSION TITLE
Does Rust have game engine?

### DISCUSSION THREAD

**Discussion by bintangyosua** [utilForever/game-developer-roadmap#46] [Q&A]
Im still confused currently. Yeah, rust's a cool language, it has so many high level safetyness 💖 and im going to about use it. But, at what cost?

**Comment by dreams3212** [comment]
I have a vr game that I want to create but I don't have developers that can help I was wondering if anyone is willing to help me out?

**Reply by utilForever** [reply]
Hmm... What do you want to make?

**Reply by dreams3212** [reply]
so you know ghost of tabor and tarkov well mix both of them and make it way better if u got discord u can add me and we can talk about it more I already have a list of things I want to do  discord - dreams1245

**Reply by dreams3212** [reply]
like a roadmap of how to make it and what I want in the game
